internal/db/schema/migrations/oss/postgres/1000/01_ldap.up.sql

```diff
@@ -232,6 +232,28 @@ comment on table auth_ldap_group_entry_search is
 'auth_ldap_group_entry_search entries specify the required parameters to find '
 'the groups a user is a member of';
 
+create function auth_ldap_method_group_search() returns trigger 
+as $$
+declare
+  n integer;
+begin
+  if new.enable_groups = true and  new.use_token_groups = false then 
+    select into n count(*) from auth_ldap_group_entry_search where ldap_method_id = new.public_id;
+     if n < 1 then 
+        raise exception 'During % of auth_ldap_method public_id=% must have a configure group_dn when enable_groups = true and use_token_groups = false',tg_op,new.public_id;
+      end if;
```

Comment:
The whitespace for these 3 lines should probably be consistent?

Reply:
done.